modules/bibdk_reservation/includes/bibdk_reservation.messages.inc

```diff
@@ -71,15 +73,11 @@ function bibdk_reservation_get_action_links($ids, BibdkWork $bibdk_work_entity)
   }
 
   if ($manifestation->hasDirectLink() && $manifestation->getRestrictedExtendedUse()) {
-    if (!user_is_logged_in()) {
-      // not logged in, do log on to see if your favorite library gives access
-    } else {
-      $links['wrapper']['item_list_link'] = bibdk_saou_link_to_soau_ressource($manifestation);
-    }
-  } else {
-    if ($manifestation->hasDirectLink()) {
-      $links[]['#markup'] = _ting_openformat_render_direct_link($manifestation->getInfotext(), $manifestation);
-    }
+    $links[]['#markup'] = t('Only restricted access', array(), array('context' => 'bibdk_reservation'));
+    $links[]['#markup'] = _ting_openformat_render_direct_link($manifestation->getInfotext(), $manifestation);
```

Comment:
overskriver du ikke ovenstående her? Mente du $links[]['#markup'] .= ......

Reply:
Nej, den adder jo bare endnu en ['#markup] med [], Det virker i hvert fald. Vil .= ikke fejle - har ikke lige testet? Men hvis det er dårlig Drupal-kode at have to gange #markup, så må jeg lige gøre det på en anden måde.

Reply:
åhhhh - du har ret.

Reply:
På en eller anden måde lykkedes det mig at overse den første []... Selvom jeg skrev den i min kommentar... *suk*